src/components/app-configure/chain-management/AppChainManagementList.vue

```diff
@@ -188,4 +207,17 @@ td {
   font-weight: 500;
   text-overflow: ellipsis;
 }
+
+.btn-group {
+  white-space: nowrap;
+}
+
+.btn-group .btn {
+  display: inline-block;
+  float: none;
```

Comment:
Why do we need this float value?

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/295/commits/0d69c93b4cb98728da38188bdb528894405157d3)